dist/index.html

```diff
@@ -1 +1,18 @@
-<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/><title>To do list</title><script defer="defer" src="/To_Do_list/bundle.js"></script></head><body><div id="to_do_list"><ul id="ulTask"><li><h4>Today's to do</h4></li><li><input placeholder="Add to your list..." id="task" name="task"></li></ul></div></body></html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <title>To do list</title>
+<script defer src="/To_Do_list/bundle.js"></script></head>
+<body>
+    <div id="to_do_list">
+        <ul id="ulTask">
+            <li><h4>Today's to do</h4></li>
+            <li><form id="form"><input type="text" placeholder="Add to your list..." id="task" name="task"><button type="submit" id="addTask"><i class="fab fa-pushed"></i></button></form></li>
```

Comment:
- [ ] Your project is apparently **not** fully completed,,seems like you mixed up the branches . The **demo link** is showing current project but the branch `crud` which you propose to merge does not contain most required features implemented. It currently cannot **create**,  **read**,  **update** and **delete**. Kindly ensure to correct this issue

**Project on Branch `crud` looks like this:**
![image](https://user-images.githubusercontent.com/12745474/130098433-fc98c787-a419-4b2c-b2c1-3ffdf06d00b4.png)

**Demo link project looks like this:**
![image](https://user-images.githubusercontent.com/12745474/130099410-e2811515-73b0-4f0d-901d-04ecd63daed2.png)

- [ ] Demo link project: Your input element is accepting empty fields. Kindly ensure to validate this input to check that the input is not empty before data is stored.

Reply:
@rloterh I ran the project locally using "npm run start" on crud branch and everything is working fine on my side. it's exactly like the demo link.
Form validation is not part of the requirements of the project.